<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ersteGrund_gui</class>
 <widget class="QDialog" name="ersteGrund_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>1. &quot;geod√§tische Hauptaufgabe&quot;</string>
  </property>
  <widget class="QPushButton" name="umrechnen">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>umrechnen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Testwerte">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Testwerte</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="s12">
      <property name="text">
       <string>s12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_s12"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="t12">
      <property name="text">
       <string>t12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_t12"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>199</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="Rechtswert">
      <property name="text">
       <string>Rechtswert y&lt;sub&gt;2&lt;/sub&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="lineEdit_y2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="Hochwert">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hochwert x&lt;span style=&quot; vertical-align:sub;&quot;&gt;2	&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QLineEdit" name="lineEdit_x2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="x1">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>61</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_x1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="y1">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>32</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>y1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_y1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
